esp-link/cgiwifi.c

```diff
@@ -342,7 +378,6 @@ static bool parse_ip(char *buff, ip_addr_t *ip_ptr) {
   return false;
 }
 
-#define DEBUGIP
```

Comment:
Moved it to user_config.h
On Sep 12, 2015 12:04 AM, "Thorsten von Eicken" <notifications@github.com>
wrote:

> In esp-link/cgiwifi.c
> <https://github.com/jeelabs/esp-link/pull/47#discussion_r39334256>:
>
> > @@ -342,7 +378,6 @@ static bool parse_ip(char *buff, ip_addr_t *ip_ptr) {
> >    return false;
> >  }
> >
> > -#define DEBUGIP
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jeelabs/esp-link/pull/47/files#r39334256>.
>